Updates to 1108 RN (#1459)
astronomer/docs: astro/release-notes.md

See [Transfer a Deployment to another Workspace](deploy-code.md#transfer-a-deployment-to-another-workspace).

### Additional improvements 
 

> - The Kubernetes API of AWS data planes are no longer exposed to the public internet. The allowlist is limited to control plane IPs. New clusters will be created with this configuration, while all existing clusters will be updated by end of next week.

Reviewer: ```suggestion
- The Kubernetes API is no longer exposed to the public internet on AWS data planes. The allowlist is limited to control plane IPs. New clusters will be created with this configuration, while all existing clusters will be updated by end of next week.
```

> - The Kubernetes API of your AWS data plane is no longer exposed to the public internet. The allowlist is locked down to Control Plane IPs. New clusters will be created with this configuration, while all existing clusters will be updated by end of next week.

Reviewer: ```suggestion
- The Kubernetes API of AWS data planes are no longer exposed to the public internet. The allowlist is limited to control plane IPs. New clusters will be created with this configuration, while all existing clusters will be updated by end of next week.
```
